source/connect/favorite-connections/import-export-ui.txt

```diff
@@ -0,0 +1,41 @@
+.. _compass-export-import-ui-landing:
+
+=========================================================
+Export and Import Favorite Connections on |compass-short|
```

Comment:
[nit]

Generally, I think we say you do things "in" Compass, rather than "on".